src/badguy/dispenser.cpp

```diff
@@ -224,8 +226,13 @@ Dispenser::collision(GameObject& other, const CollisionHit& hit)
 }
 
 void
-Dispenser::active_update(float )
+Dispenser::active_update(float dt_sec)
 {
+  if (m_gravity)
+  {
+    BadGuy::active_update(dt_sec);
+    m_physic.enable_gravity(true);
```

Comment:
Does `Badguy::active_update` disable gravity again?

Reply:
Yes, it does.

Reply:
Yeah, I know, but is it necessary to call `enable_gravity` again after calling `active_update`?

Reply:
It appears not, so it's been removed.